R/fill.R

```diff
@@ -44,51 +43,76 @@
 #' # Inner doesn't fill outer
 #' if (interactive()) browsable(tagz)
 #'
-#' tagz <- asFillContainer(tagz)
-#' tagz <- asFillItem(tagz, .cssSelector = "#inner")
+#' tagz <- bindFillRole(tagz, container = TRUE)
+#' tagz <- bindFillRole(tagz, item = FALSE, .cssSelector = "#inner")
 #'
 #' # Inner does fill outer
 #' if (interactive()) browsable(tagz)
 #'
-asFillContainer <- function(x, ..., height = NULL, width = NULL, asItem = FALSE, .cssSelector = NULL) {
-  if (!inherits(x, "shiny.tag")) {
-    return(throwFillWarning(x))
-  }
+bindFillRole <- function(x, ..., item = FALSE, container = FALSE, overwrite = FALSE, .cssSelector = NULL) {
 
   ellipsis::check_dots_empty()
```

Comment:
Are you using the `...` as the equivalent of Python's `*`?

Reply:
Yep